dev-requirements.txt

```
@@ -5,7 +5,7 @@
 #
 # ref: https://github.com/jupyterhub/chartpress
 #
-chartpress>=1.1.0
+chartpress==2.*
```

```suggestion
chartpress>=2.1,<3
```

Since we want to use tbump